pkgs/development/python-modules/numpy/default.nix

```diff
@@ -54,6 +56,23 @@ in buildPythonPackage rec {
     # Patching of numpy.distutils is needed to prevent it from undoing the
     # patch to distutils.
     ./numpy-distutils-C++.patch
```

Comment:
We probably don't want to include the other patches based on distutils

Reply:
That was already in the derivation. I assumed it was necessary until numpy is updated and switched to meson.

Reply:
I was more hinting that we need to close the list here before adding the other patches.

Reply:
Oh, you’re right. I completely missed the conditional. I understand now and will push a fix for that piece.

Reply:
Yes, I meant talking about the whole bunch, as they don't seem useful/needed for our gcc linux builds.

Reply:
I made each of the patches conditional.

Reply:
This got uglier than I expected.  Cleanup PR: https://github.com/NixOS/nixpkgs/pull/244461